rpm/libhybris.spec

```diff
@@ -1,3 +1,6 @@
+%define USE_MALI %{getenv:USE_MALI}
```

Comment:
This has been done in the following approach instead:
https://github.com/mer-hybris/droid-hal-device/commit/d32f25ba40f1149379b3c5a18c0965d8a54e45b8
https://github.com/libhybris/libhybris/pull/360/commits/35bbc6ca5234a7cd6d2e9264bf03aab10fab7844

Closing this PR